x-pack/test/apm_api_integration/tests/alerts/transaction_error_rate.spec.ts

```diff
@@ -0,0 +1,169 @@
+        .service({ name: 'opbeans-node', environment: 'production', agentName: 'node' })
+        .instance('instance');
+      const events = timerange('now-15m', 'now')
+        .ratePerMinute(1)
+        .generator((timestamp) => {
+          return [
+            opbeansJava
+              .transaction({ transactionName: 'tx-java' })
+              .timestamp(timestamp)
+              .duration(100)
+              .failure(),
+            opbeansJava
+              .transaction({ transactionName: 'tx-java' })
+              .timestamp(timestamp)
+              .duration(200)
+              .success(),
+            opbeansNode
+              .transaction({ transactionName: 'tx-node' })
+              .timestamp(timestamp)
+              .duration(400)
+              .success(),
+          ];
+        });
+      await synthtraceEsClient.index(events);
+    });
+
+    after(async () => {
+      await synthtraceEsClient.clean();
+      await supertest.delete(`/api/alerting/rule/${ruleId}`).set('kbn-xsrf', 'foo');
+      await supertest.delete(`/api/actions/connector/${actionId}`).set('kbn-xsrf', 'foo');
+      await esDeleteAllIndices([INDEX_NAME]);
+      await es.deleteByQuery({
+        index: '.alerts*',
+        query: { term: { 'kibana.alert.rule.uuid': ruleId } },
+      });
+      await es.deleteByQuery({
+        index: '.kibana-event-log-*',
+        query: { term: { 'kibana.alert.rule.consumer': 'apm' } },
+      });
+    });
+
+    describe('create alert with transaction.name group by', () => {
+      before(async () => {
+        actionId = await createIndexConnector({
+          supertest,
+          name: 'Transation error rate API test',
+          indexName: INDEX_NAME,
+        });
+        const createdRule = await createApmRule({
+          supertest,
+          ruleTypeId: ApmRuleType.TransactionErrorRate,
+          name: 'Apm error rate duration',
+          params: {
+            threshold: 50,
+            windowSize: 5,
+            windowUnit: 'm',
+            transactionType: 'request',
+            serviceName: 'opbeans-java',
+            environment: 'production',
+            groupBy: [
+              'service.name',
+              'service.environment',
+              'transaction.type',
+              'transaction.name',
+            ],
+          },
+          actions: [
+            {
+              group: 'threshold_met',
+              id: actionId,
+              params: {
+                documents: [{ message: 'Transaction Name: {{context.transaction.name}}' }],
+              },
+              frequency: {
+                notify_when: 'onActionGroupChange',
+                throttle: null,
+                summary: false,
+              },
+            },
+          ],
+        });
+        expect(createdRule.id).to.not.eql(undefined);
+        ruleId = createdRule.id;
+      });
+
+      it('checks if alert is active', async () => {
+        const executionStatus = await waitForRuleStatus({
```

Comment:
Like mentioned above: can you also verify that the group-by params are added to the alert as expected?

Reply:
done